File: programers_Lv2/JadenCase 문자열 만들기.py
def solution(s):
    char_list = list(s.lower())

    char_list[0] = char_list[0].upper()

    for i in range(len(char_list)):
        if char_list[i] != ' ' and char_list[i-1] == ' ':
            char_list[i] = char_list[i].upper()

    return ''.join(char_list)

print(solution("  a  3peoPle 3peoPle  unFollowed me    me unFollowed"))
print(solution('    aaaaa aaa aaaaaaa a aa a'))
print(solution(" 3people unFollowed me"))
print(solution("aaa a   aaa    "))
print(solution("for the last week"))

File: programers_Lv2/JadenCase 문자열 만들기.js
function solution(s) {
  var answer = [...s].map((el) => el.toLowerCase());

  answer[0] = answer[0].toUpperCase();

  for (let i = 0; i < answer.length; i++) {
    if (answer[i] !== " " && answer[i - 1] === " ") {
      answer[i] = answer[i].toUpperCase();
    }
  }

  return answer.join("");
}

console.log(solution("for the last week"));
